RevenueCatUI/CustomerCenter/Views/CompatibilityLabeledContent.swift

```diff
@@ -66,8 +66,8 @@ import SwiftUI
 @available(watchOS, unavailable)
 struct CompatibilityLabeledContent<Label: View, Content: View>: View {
 
-    let label: () -> Label
-    let content: () -> Content
+    @ViewBuilder let label: () -> Label
+    @ViewBuilder let content: () -> Content
```

Comment:
Thanks for adding this back! I don't know how it happened (maybe the reversed commits?) but I had already made this change in #5071. However, that PR now appears empty 🤯. Anyway, thanks again!